tests/components/mqtt/test_humidifier.py

```diff
@@ -87,7 +87,9 @@ async def async_turn_on(hass: HomeAssistant, entity_id: str = ENTITY_MATCH_ALL)
     """Turn all or specified humidifier on."""
     data = {ATTR_ENTITY_ID: entity_id} if entity_id else {}
 
-    await hass.services.async_call(DOMAIN, SERVICE_TURN_ON, data, blocking=True)
+    await hass.services.async_call(
+        HUMIDIFIER_DOMAIN, SERVICE_TURN_ON, data, blocking=True
+    )
```

Comment:
```suggestion
    await hass.services.async_call(
        humidifier.DOMAIN, SERVICE_TURN_ON, data, blocking=True
    )
```

I would like to use the platform import instead. I think this used at other places as well.

I could open a PR for that...

Reply:
Thanks, I'll open a PR